rpc/codec/server.go

```diff
@@ -24,11 +24,17 @@ package codec
 import (
 	"bufio"
 	"bytes"
+	"crypto/tls"
 	"fmt"
 	"io"
 	"net/rpc"
 
+	"github.com/cockroachdb/cockroach/base"
+	cockroach_proto "github.com/cockroachdb/cockroach/proto"
```

Comment:
doesn't really concern your PR, but I've noticed how `etcd/raft` named its proto package `raftpb`, so if it's not mandated by style to have our proto package called `proto`, we could just rename both the `proto` folder and the contained package to `roachpb` (or even shorter, `pb`), which would allow us to always use the canonical import path.